### PR TITLE
[+] Types Support - Support BIGINT field type

### PR DESCRIPTION
**Pull request by lucasscariot**

### CODE REVIEW

**Comment by lucasscariot** [comment]
[(0.5) Users want Forest to support BIGINT fields](https://trello.com/c/E5woMH5p/1506-0-5-users-want-forest-to-support-bigint-fields)

File: adapters/sequelize.js [hunk 1]
```diff
@@ -103,9 +104,18 @@ module.exports = function (model, opts) {
...
+        isCompositePrimary = true;
+        idField = 'forestCompositePrimary';
+      }
+
       return {
         name: model.name,
-        idField: _.keys(model.primaryKeys)[0],
+        idField: idField,
+        isCompositePrimary: isCompositePrimary,
         fields: fields
```
Comment (arnaudbesnier) [inline adapters/sequelize.js]: I think that theses changes are not related to the field type support. Don't you think?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- BIGINT - BIGINT are supported in creation, edition & read.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: - Types Support - Support BIGINT field type.
